src/cudamatrix/cu-sparse-matrix.cc

```diff
@@ -57,7 +57,8 @@ template <typename Real>
 Real CuSparseMatrix<Real>::Sum() const {
 #if HAVE_CUDA == 1
```

Comment:
Please rebase against the latest master and test.  there should be lines here:
  if (NumElements() == 0)
    return 0.0;
Please test by logging into a GPU machine and going 'make test' in src/ - this will test more thoroughly.